roles/pbx/tasks/freepbx.yml

```diff
@@ -124,14 +124,14 @@
     state: file
     mode: "{{ item.mode }}"
   with_items:
-  - { path: '/etc/freepbx.conf', mode: 0664 }
-  - { path: '/var/log/asterisk/freepbx.log', mode: 0666 }
+  - { path: '/etc/freepbx.conf', mode: u=rw,g=r,o=r }
```

Comment:
Is this `/etc/freepbx.conf` change from 664 to 644 on purpose?

(I don't know the official ownership / permissions recommendations!)

Reply:
Not sure if official nginx docs exist for freepbx.. I'll explain in greater detail.